package.json

```diff
@@ -53,7 +53,7 @@
     "chalk": "2.3.0",
     "cli-table2": "0.2.0",
     "command-exists": "1.2.2",
-    "fs-extra": "4.0.2",
+    "fs-extra": "^4.0.2",
```

Comment:
We don't like to do this sort of pinning as it lets packages get upgraded to minor versions which often contain new bugs or breaking changes.

Reply:
Yes, agreed.  Hard pinned versions only around here.  Too easy for users to get unintentional updates from some package that gets careless about breaking changes.

Reply:
@zenweasel done.